docs/source/distributions.rst

```diff
@@ -377,10 +377,68 @@ TransformModule
     :undoc-members:
     :show-inheritance:
 
-Constraints
-~~~~~~~~~~~
-.. automodule:: pyro.distributions.constraints
-    :members:
-    :member-order: bysource
-    :undoc-members:
-    :show-inheritance:
+Transform Factories
```

Comment:
As commented below, let's try to ensure these appear in some docs sidebar somewhere. No need to do so in this PR, we can tweak docs in follow-up PRs.